src/test/cypress/integration/exercises/modeling/ModelingExerciseManagement.spec.ts

```diff
@@ -109,8 +107,14 @@ describe('Modeling Exercise Management Spec', () => {
             createModelingExercise.setPoints(points);
             createModelingExercise.save();
             cy.visit(`/course-management/${course.id}/exercises`);
-            cy.get('tbody > tr > :nth-child(2)').should('contain.text', newTitle);
-            cy.get('tbody > tr > :nth-child(6)').should('contain.text', points.toString());
+            cy.get('#modeling-exercise-card-' + modelingExercise.id)
+                .children()
+                .eq(1)
+                .should('contain.text', newTitle);
+            cy.get('#modeling-exercise-card-' + modelingExercise.id)
+                .children()
+                .eq(5)
+                .should('contain.text', points.toString());
```

Comment:
This would theoretically be possible and not require an apollon release as this is a artemis component. It would have to be dynamic ids since theres usually a lot of these fields within a page.. But yes in principle this would be possible.

Reply:
I guess ill try to get this one done.